completions/_dust

```diff
@@ -37,6 +37,12 @@ _dust() {
 '--output-format=[Changes output display size. si will print sizes in powers of 1000. b k m g t kb mb gb tb will print the whole tree in that size.]:FORMAT:(si b k m g t kb mb gb tb)' \
 '-S+[Specify memory to use as stack size - use if you see\: '\''fatal runtime error\: stack overflow'\'' (default low memory=1048576, high memory=1073741824)]:STACK_SIZE: ' \
 '--stack-size=[Specify memory to use as stack size - use if you see\: '\''fatal runtime error\: stack overflow'\'' (default low memory=1048576, high memory=1073741824)]:STACK_SIZE: ' \
+'-M+[File'\''s data was last modified less than, more than or exactly <mtime>*24 hours ago. When dust figures out how many 24-hour periods ago the file was last modified, any fractional part is ignored, so to match -mtime +1, a file has to have been modified at least two days ago.]: : ' \
```

Comment:
Consider making the description more easily readable


```suggestion
'-M+[Sort by time modifed, include only files modified +/-<mtime> days before/after, days are rounded down, prefix \'\-\' for newer, \'\+\' for older. : : ' \
```

Reply:
In my opinion, Filter by time mofied is better than Sort by time modifed because this parameter is used for filtering, not sorting. right? :)

Reply:
Sure, you can rephrase it as you wish, just pointed that its a bit too long :)

Reply:
Indeed, that description is a bit too long. I rephrased it . I will be very happy if you can help me review the code again. :)